pcap.c

```diff
@@ -2104,22 +2109,21 @@ pcap_offline_filter(const struct bpf_program *fp, const struct pcap_pkthdr *h,
  * from the library but being truncated).
  */
 #ifdef HAVE_VERSION_H
-#include "version.h"
-#else
-static const char pcap_version_string[] = "libpcap version 1.x.y";
+#include "..\..\version.h"
```

Comment:
No, that won't work on UN\*X - version.h is in the same directory as the source code (*and* \\, of course, is not a pathname separator...).